igvm/commands.py

```diff
@@ -420,3 +536,60 @@ def vm_rename(vm_hostname, new_hostname, offline=False):
         )
 
     vm.rename(new_hostname)
+
+
+def _get_vm(hostname, ignore_reserved=False):
+    """Get a server from Serveradmin by hostname to return VM object
+
+    The function is accepting hostnames in any length as long as it resolves
+    to a single server on Serveradmin.
+    """
+    conditions = [ExactMatch(hostname)]
+    if hostname.endswith('.ig.local'):
+        conditions.append(ExactMatch(hostname[:-len('.ig.local')]))
```

Comment:
We should add a todo to remove this when we have all servers renamed properly

Reply:
This was actually to find the hosts without FQDN when the search is with FQDN.  We don't need this anymore.  So I am removing it, and putting the `TODO` to the place we append `.ig.local`.

Reply:
I prepared #18 for this.